endpoints/openrtb2/auction.go

```diff
@@ -997,32 +1002,41 @@ func validateSChains(sChains []*openrtb_ext.ExtRequestPrebidSChain) error {
 	return err
 }
 
-func (deps *endpointDeps) validateEidPermissions(prebid *openrtb_ext.ExtRequestPrebidData, requestAliases map[string]string) error {
+func (deps *endpointDeps) validateEidPermissions(prebid *openrtb_ext.ExtRequestPrebidData, requestAliases map[string]string, isDebugEnabled bool) (error, []error) {
+	warnings := make([]error, 0)
```

Comment:
This will always allocate memory. Please use `var errs []error` to avoid the memory allocation when there are no errors, assuming most requests will not have an error / warning. Calls to `append` will allocate memory when needed.